### ISSUE TITLE
How do you build and run the desktop app?

### ISSUE THREAD

**Issue by InimicalPart** [requestly/requestly-desktop-app#66]
## Summary

I've been gnawing at this for hours and I cannot figure out how to successfully build and run the Requestly desktop app so that you can skip the restrictions set in the free pricing plan.

![image](https://github.com/requestly/requestly-desktop-app/assets/37552123/882ebeee-5760-481c-8763-05f86a7dfa04)
<sup><sub>(Source: https://app.requestly.io/pricing)</sub></sup>

The instructions provided in [the README](https://github.com/requestly/requestly-desktop-app?tab=readme-ov-file#setup) are really difficult to follow and some steps are only designed to work for Linux/MacOS, but not Windows.

Could someone please tell me *exactly* how to build and run the desktop app so that there are no restrictions on the free plan?
Thank you in advance!

**Comment by sagarsoni7** [comment]
Thanks @InimicalPart for logging this issue. Have you used Electron before?
Also could you please ensure that the node & npm version matches.

**Comment by InimicalPart** [comment]
Sorry, I didn't get a notification about your reply. Yes I am familiar with Electron, node and npm versions match as well

**Comment by sagarsoni7** [comment]
@InimicalPart  - Great then let me help you step by step. The desktop app doesn't have it's own UI.
First you need to ensure that the Requestly Dashboard UI is running successfully on your localhost.

You need to clone and run this repo first: https://github.com/requestly/requestly/

**Comment by sagarsoni7** [comment]
@InimicalPart - Meanwhile you could also request a free trial of Requestly. Drop hi at contact@requestly.io

**Comment by InimicalPart** [comment]
I actually needed Requestly for a small project of mine and needed the modify response feature, but I managed to find a way to do what I had to do without Requestly.

For those that are also trying to use Requestly in a project of theirs, and are wondering what I did, I used mitmproxy with a custom made Python addon to edit the requests and responses that went through it.

Since I no longer need this done, I will close the issue. Thanks for trying to help out!
